packages/hub/plugin-loader.js

```diff
@@ -24,7 +24,8 @@ const featureTypes = [
   'authenticators',
   'middleware',
   'messengers',
-  'code-generators'
+  'code-generators',
+  'queues'
```

Comment:
I think we're not using this? It would be needed if we were using the plugin-loader to discover queues from arbitrary plugins, but I think the hub is just hard coded to use the one implementation we have, and I think that's good.